AdminWebsite/AdminWebsite/ClientApp/src/app/shared/footer/footer.component.scss

```diff
@@ -0,0 +1,11 @@
+.app-version {
+  position: relative;
+  top: 0;
+  text-align: center;
+  padding: 10px;
+  margin: 0;
+}
+
+.no-margin {
```

Comment:
You can achieve the same thing using `govuk-!-margin-bottom-0`

Reply:
done